common/OpTestSystem.py

```diff
@@ -394,11 +394,18 @@ def detect_target(self, target_state, reboot):
         self.console.enable_setup_term_quiet()
         sys_pty = self.console.get_console()
         self.console.disable_setup_term_quiet()
-        sys_pty.sendcontrol('c')
+        sys_pty.sendline()
```

Comment:
the sendline() could be risk though, as we could be sitting in pettiboot and this could boot the system and thus change the state?

Reply:
Good point; perhaps instead we can do an `x` here which should still feed input into pexpect. May get a stray `x` in the shell but the current detection code should deal with that.

Reply:
Maybe
```
Esc[K | Erase Line: 	Clears all characters from the cursor position to the end of the line (including the character at the cursor position).
```
http://ascii-table.com/ansi-escape-sequences.php

Reply:
IIRC originally* I had `ctrl-l` i think, which is for redrawing everything.. any reason that wouldn't work here?

* for certain values of 'originally'

Reply:
Sometimes redrawing nothing does not put anything in the buffer, need to produce something new

Reply:
Yep, earlier versions of https://github.com/open-power/op-test-framework/pull/362 changed this to Ctrl-L but it wasn't 100% reliable